daemon/volumes.go

```diff
@@ -29,7 +29,7 @@ type mountPoint struct {
 	RW          bool
 	Volume      volume.Volume `json:"-"`
 	Source      string
-	Relabel     string
+	Mode        string
```

Comment:
So that means we would continue to store the data forever as Relabel, but read it in as Mode?